src/xdocs/config_naming.xml

```diff
@@ -369,11 +369,30 @@
           An example of how to configure the check for names that begin with
           a lower case letter, followed by any letters or digits is:
         </p>
+        <p>Configuration:</p>
```

Comment:
"Configuration: to require format on two or more characters started by lowercase"

Reply:
@romani The description of the configuration is right above. Aren't we just restating the same thing?

Reply:
I missed it, my bad.
this item should be skipped.